### PR TITLE
Add package crypto-policies version 20200619

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add package crypto-policies version 20200619

###### Change Log  

- Add package crypto-policies version 20200619

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Local Build.

BuddyBuild: 
- [AMD](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293089&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=1630)
- [ARM](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293090&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=d723e8ff-9fe0-5a1d-bed2-08b8dc44a520&l=20865)

### CODE REVIEW

File: SPECS-EXTENDED/crypto-policies/crypto-policies.spec [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}-scripts = %{version}-%{release}
+Requires:       dracut
+Requires:       grubby
+
+%description -n fips-mode-setup
+The package provides a tool to enable or disable the system FIPS mode.
+
+%prep
+%setup -q -n fedora-crypto-policies-f32-updates
```
Comment (Sumynwa) [inline SPECS-EXTENDED/crypto-policies/crypto-policies.spec:57]: Care to use %autosetup.

**Comment by Sumynwa** [comment]
LGTM

File: SPECS-EXTENDED/crypto-policies/crypto-policies.spec (hunk 1 above)
Comment (Sumynwa) [reply SPECS-EXTENDED/crypto-policies/crypto-policies.spec:57]: Just a nit-pick

**Review by Sumynwa — APPROVED** [review]

**Comment by mfrw** [comment]
Build post rebase: [build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=319926&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=1644)

**Pull request merged**